assets/src/realtime.d.ts

```diff
@@ -84,6 +84,10 @@ export type RouteStatus = "on_route" | "laying_over" | "pulling_out"
 
 export type RunId = string
 
+export interface ByRunId<T> {
+  [runId: string]: T
```

Comment:
would `[runId: RunId]: T` work?

Reply:
One would thing so, but sadly it would not:
```
src/realtime.d.ts:88:4 - error TS1336: An index signature parameter type cannot be a type alias. Consider writing '[runId: string]: T' instead.
```

Reply:
yeah, figured you probably would have tried that...